aisynphys/qc.py

```diff
@@ -140,25 +140,25 @@ def pulse_response_qc_pass(post_rec, window, n_spikes, adjacent_pulses):
         [failures[k].append('Spikes detected within 8ms of the response window') for k in failures.keys()]
 
     # Check holding potential is appropriate for each sign
-    ex_limits = [-85e-3, -45e-3]
-    in_limits = [-60e-3, -45e-3]
+    ex_limits = [-85e-3, -50e-3]
```

Comment:
Should we put the hyperpolarized bound at -80? I honestly don't remember why this was set different from the other one